src/index.js

```diff
@@ -17,7 +17,9 @@ function App({listings, listingCategoryIcons}) {
       <Page>
         <Routes>
           <Route path="/about" element={<About />} />
-          <Route path="/suggest" element={<SuggestUpdate />} />
+          <Route path="/suggest" element={<SuggestUpdate activeIndex={0} />} />
```

Comment:
This may seem redundant, but it does actually achieve the UX the client asked for (for better or for worse.)